.github/workflows/tests.yml

```diff
@@ -89,12 +91,10 @@ jobs:
         with:
           distribution: "corretto"
           java-version: 17
-      - name: Validate Indentations
-        run: ./gradlew spotlessCheck
       - name: Validate Build
-        run: ./gradlew build -x test
+        run: ./gradlew server:spotlessCheck test-utils:spotlessCheck server:build -x test
       - name: Tests
-        run: ./gradlew server:test server:e2e server:build
+        run: ./gradlew server:test server:e2e
```

Comment:
These changes do not reduce our pipeline coverage; however, they reduce the build time by 59 seconds.